### DISCUSSION TITLE
Motion Detecting w/ Visual Builder, Touch Screen

### DISCUSSION THREAD

**Discussion by cstan22** [gopro/labs#1054] [General]
I'm hitting a wall. Ended up with something like this. After scanning the QR code, it records immediately, and after it ends capture, it gets buggy. Sometimes it recognizes motion again. Sometimes not. When it does, it's often pretty delayed in starting / ending captures. The script will cycle through printing "Recycle" and "No motion", leading me to think that there's some bug that stops it from activating motion detection again after each round. The upload feature works well, and it even started detecting motion again after the device finished clearing the SD, but the touch screen stayed frozen on "clearing SD", so it wasn't obvious that it was working at first.

Using the old-style command language, !SM5-5H3!R works best, but then I can't add a time-of-day upload event.

Any thoughts are appreciated!

<img width="424" alt="2024-09-09_11-55-57" src="https://github.com/user-attachments/assets/8e3bfa2e-79f8-4dd0-8b31-f510871ecde0">

**Comment by dnewman-gpsw** [comment]
Closing as a duplicate
